faults/load/k6-loadgen/engine.yaml

```
@@ -0,0 +1,41 @@
+---
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: nginx-chaos
+  namespace: default
+spec:
+  engineState: 'active'
+  chaosServiceAccount: litmus-admin
+  experiments:
+    - name: k6-loadgen
+      spec:
+        components:
+          env:
+            # set chaos duration (in sec) as desired
+            - name: TOTAL_CHAOS_DURATION
+              value: "30"
+
+            # Interval between chaos injection in sec
+            - name: CHAOS_INTERVAL
+              value: "30"
+
+            # Period to wait before and after injection of chaos in sec
+            - name: RAMP_TIME
+              value: "0"
+
+            # Provide the secret name of the JS script
+            - name: SCRIPT_SECRET_NAME
+              value: "k6-script"
+
+            # Provide the secret key of the JS script
+            - name: SCRIPT_SECRET_KEY
+              value: "script.js"
+
+            # Provide the image name of the helper pod
+            - name: LIB_IMAGE
+              value: "ghcr.io/grafana/k6-operator:latest-runner"
+
+            # Provide the image pull policy of the helper pod
+            - name: LIBImagePullPolicy
```

This is incorrect, please change to
```suggestion
            - name: LIB_IMAGE_PULL_POLICY
```